lib/lz4.c

```diff
@@ -178,6 +178,12 @@
 #define unlikely(expr)   expect((expr) != 0, 0)
 #endif
 
+/* Should the alignment test prove unreliable, for some reason,
+ * it can be disabled by setting LZ4_ALIGN_TEST to 0 */
+#ifndef LZ4_ALIGN_TEST  /* can be externally provided */
+# define LZ4_ALIGN_TEST 1
+#endif
```

Comment:
This is a very, very controversial decision! It will be quite difficult to get people using machines and compilers from the first list https://github.com/lz4/lz4/issues/827#issuecomment-719983334 to specify this definition at compilation.